### PR TITLE
add docker-compose deploy and the docker backup deal tips

### PR DESCRIPTION
**Pull request by wxcsdb88**
1. add the docker-compose deploy files docker-compose.yml
2. add the docker deploy backup deals in docker-deploy-backup.md
3. define ENV in Dockfile

---
This change is [Reviewable](https://reviewable.io/reviews/star7th/showdoc/148)

### CODE REVIEW

**Comment by wxcsdb88** [comment]
With the `cron` backup deals,  we needn\`t use docker `volume` to backup the container data in `sqlite.db.php` and uploads files!

File: docker-delpoy-backup.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+## docker-compose op
+### create docker container and start
+change the dir to showdoc root dir and execute followings:
+```
+sudo docker-compose up
+# sudo docker-compose up -f docker-compose.yml
+```
+
+### start
+```
```
Comment (czok) [inline docker-delpoy-backup.md]: sudo docker-compose up

File: docker-delpoy-backup.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+sudo docker-compose start
+```
```
Comment (czok) [inline docker-delpoy-backup.md]: # sudo docker-compose up -f docker-compose.yml

**Comment by star7th** [comment]
时隔那么久，官方已经对docker使用进行了封装，一个命令行脚本就可以自动安装。https://www.showdoc.cc/help?page_id=828455960655160

**Pull request closed without merging**